server/utils/characteristicsUtils.ts

```diff
@@ -17,11 +23,22 @@ export const characteristicsPairToCharacteristics = (
     .map(({ propertyName }) => (roomCharacteristicMap[propertyName] ? propertyName : null) as Cas1SpaceCharacteristic)
     .filter(Boolean)
 
-export const characteristicsBulletList = (characteristics: Array<Cas1SpaceCharacteristic>): string =>
-  `<ul class="govuk-list govuk-list--bullet">${characteristics
-    .filter(characteristic => roomCharacteristicMap[characteristic])
-    .map(characteristic => `<li>${roomCharacteristicMap[characteristic]}</li>`)
-    .join('')}</ul>`
+export const characteristicsBulletList = (
+  requirements: Array<Cas1SpaceCharacteristic | PlacementCriteria>,
+  options: { labels?: Record<string, string>; noneText?: string } = {},
+): string => {
+  const { labels = placementCriteriaLabels, noneText = `<span class="text-grey">None</span>` } = options
 
-export const characteristicsInlineList = (criteria: Array<Cas1SpaceCharacteristic>): string =>
-  joinWithCommas(criteria.map(characteristic => roomCharacteristicMap[characteristic].toLowerCase()))
+  const listItems = Object.keys(labels).filter(key => ((requirements || []) as Array<string>).includes(key))
+
+  return listItems.length
+    ? `<ul class="govuk-list govuk-list--bullet">${listItems.map(key => `<li>${labels[key]}</li>`).join('')}</ul>`
+    : noneText
+}
+
+export const characteristicsInlineList = (
```

Comment:
Maybe rename to `roomCharacteristicsInLineList` as I think it can only handle room characteristics.